styles/microsoft-word/catppuccin.user.css

```diff
@@ -0,0 +1,358 @@
+/* ==UserStyle==
+@name Microsoft Word Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/microsoft-word
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/microsoft-word
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/microsoft-word/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Amicrosoft-word
+@description Soothing pastel theme for Microsoft Word
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavour "Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
```

Comment:
Dark/light mode?

Reply:
I think it wasn't possible for *all* of ms365 but I believe word does have a way, will try to implement it later.